app/views/simplified-account/settings/card-types/cardTypesCheckboxes.njk

```diff
@@ -3,6 +3,7 @@
 
   <form method="post" novalidate>
     <input id="csrf" name="csrfToken" type="hidden" value="{{csrf}}"/>
+    <input type="hidden" name="currentAcceptedCardTypeIds" value="{{ currentAcceptedCardTypeIds }}" />
```

Comment:
I've updated the filename. It's not part of the main template because it's only required under one condition and this seemed neater but happy to change if you prefer it all in one template.

Reply:
updated so there's only one template file.